agent-lib/src/main/java/org/terracotta/angela/agent/AgentController.java

```diff
@@ -389,14 +335,9 @@ public void uninstallTsa(InstanceId instanceId, Topology topology, TerracottaSer
     }
   }
 
-  public void uninstallTms(InstanceId instanceId, Distribution distribution, TmsServerSecurityConfig tmsServerSecurityConfig,
-                           String kitInstallationName, String tmsHostname) {
+  public void uninstallTms(InstanceId instanceId, Distribution distribution, String kitInstallationName, String tmsHostname) {
     TmsInstall tmsInstall = tmsInstalls.get(instanceId);
     if (tmsInstall != null) {
-      File tmcProperties = new File(tmsInstall.getKitLocation(), "/tools/management/conf/tmc.properties");
```

Comment:
same as above

Reply:
yes I will refactor... we need to keep that step if kit install dir is not supplied